Can't resolve './rxjs.imports'

Hello,

After I ran npm update(ng-dynamic-component 2.1.0 to 2.1.3) I am getting the following

I am getting webpack error when using ng-dynamic-component

ERROR in ./~/ng-dynamic-component/index.js
Module not found: Error: Can't resolve './rxjs.imports' in node_modules/ng-dynamic-component

@ ./~/ng-dynamic-component/index.js 5:0-24

Where is rxjs.imports located? I dont see it in the src tree

--- a/src/index.ts
+++ b/src/index.ts
@@ -1,2 +1,1 @@
-import './rxjs.imports';
 export * from './dynamic/index';

